### PR TITLE
Documentation Page

### PR DESCRIPTION
**Pull request by kaitj**
**Proposed changes**

This PR adds a documentation page that the moment generates information based on the docstrings of the code in this repository.

### TODO:
- [x] Update docstrings
- [x] Fix build status for PRs
- [x] Fix intersphinx mapping with NumPy

**Types of changes**
What types of changes does your code introduce? Put an `x` in the boxes that apply

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (if none of the other choices apply)

**Checklist**
Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you are unsure about any of the choices, don't hesitate to ask!

- [x] Changes have been tested to ensure that fix is effective or that a feature works.
- [x] Changes pass the unit tests
- [x] Code has been run through the `poe quality` task
- [x] I have included necessary documentation or comments (as necessary)
- [x] Any dependent changes have been merged and published

**Notes**
All PRs will undergo the unit testing before being reviewed. You may be requested to explain or make additional changes before the PR is accepted.

### CODE REVIEW

File: docs/general/main.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Overview
```
Comment (tkkuehn) [inline docs/general/main.md:1]: This seems like a really long section of text mostly describing the purpose of documentation, most of which I don't think we need. I'd keep an adjusted version of the first sentence and maybe add some examples of the kinds of operations we're talking about (or just point to the API docs).

I'd also just move whatever remains to the readme and get rid of this file/section.

File: docs/general/main.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The purpose of the `afids_utils` package is to provide commonly used utility 
+functions (written in Python) that is applicable to a various anatomical
+fiducial (AFIDS) projects.  The `afids_utils` documentation provides a
```
Comment (tkkuehn) [inline docs/general/main.md:5]:
```suggestion
`afids_utils` provides common utility for projects using anatomical
fiducial (AFIDs).
```

**Review by tkkuehn — CHANGES_REQUESTED** [review]
Mostly looks good, just would make the intro a lot more concise.

File: docs/general/main.md (hunk 1 above)
Comment (kaitj) [reply docs/general/main.md:1]: I think for now I will move the majority of this as a comment to the `README` until we have time to fully flesh it out in a separate PR.

**Review by tkkuehn — APPROVED** [review]
I would still prefer to just get rid of `general/main.md` and put its content in the README, but if you prefer this, that's ok by me and you can go ahead and merge.

**Comment by kaitj** [comment]
I'm open to that and can make the change in the AM.

**Pull request merged**